src/embed/ts-embed.ts

```diff
@@ -1020,7 +1020,9 @@ export class TsEmbed {
      * THOUGHTSPOT URL params starts with a prefix "ts-"
      * @version SDK: 1.14.0 | ThoughtSpot: 8.4.0.cl, 8.4.1-sw
      */
-    public getThoughtSpotPostUrlParams(): string {
+    public getThoughtSpotPostUrlParams(
+        additionalParams: { [key: string]: string | number } = {},
```

Comment:
can we make this param optional ?

Reply:
Isn't this already optional ? See the default assignment.